webpack-and-babel/imports.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+// import from a file located at the absolute path `/foo/bar/baz/qux.js`:
+// eslint-disable-next-line import/no-absolute-path
+import f1 from '/foo/bar/baz/qux';
+
+// import from a file named `foo.json` located in the current directory:
+import f2 from './foo';
```

Comment:
Need to include the file extension here! If it was `.js` file or something similar, Webpack would be able to resolve that, but since it's a JSON file, we need to explicitly state that.